core/iwasm/aot/aot_runtime.c

```diff
@@ -902,7 +902,7 @@ create_exports(AOTModuleInstance *module_inst, AOTModule *module,
 static bool
 clear_wasi_proc_exit_exception(AOTModuleInstance *module_inst)
 {
-#if WASM_ENABLE_LIBC_WASI != 0
+#if (WASM_ENABLE_LIBC_WASI != 0) && (WASM_ENABLE_THREAD_MGR == 0)
```

Comment:
Not sure why skip clear the exception for multi-threads?

Reply:
Restored, thanks~